cfr.json

```diff
@@ -1559,5 +1559,167 @@
     "template": "update_action",
     "targeting": "firefoxVersion >= 69 && localeLanguageCode in [\"en\",\"fr\",\"de\"] && ((currentDate|date - profileAgeCreated) / 86400000) >= 4 && ((currentDate|date - profileAgeCreated) / 86400000) < 15",
     "id": "WNP_MOMENTS_1"
+  },
+  {
+    "id": "SOCIAL_TRACKING_PROTECTION",
+    "template": "cfr_doorhanger",
+    "priority": 1,
+    "content": {
+      "layout": "icon_and_message",
+      "category": "cfrFeatures",
+      "anchor_id": "tracking-protection-icon-box",
+      "skip_address_bar_notifier": true,
+      "bucket_id": "CFR_SOCIAL_TRACKING_PROTECTION",
+      "heading_text": { "string_id": "cfr-doorhanger-socialtracking-heading" },
+      "notification_text": "",
+      "info_icon": {
+        "label": {
+          "string_id": "cfr-doorhanger-extension-sumo-link"
+        },
+        "sumo_path": "extensionrecommendations"
+      },
+      "learn_more": "social-media-tracking-report",
+      "text": { "string_id": "cfr-doorhanger-socialtracking-description" },
+      "icon": "chrome://browser/skin/notification-icons/block-social.svg",
+      "icon_dark_theme":
+      "chrome://browser/skin/notification-icons/block-social-dark.svg",
+      "buttons": {
+        "primary": {
+          "label": { "string_id": "cfr-doorhanger-socialtracking-ok-button" },
+          "action": { "type": "OPEN_PROTECTION_PANEL" },
+          "event": "PROTECTION"
+        },
+        "secondary": [
+          {
+            "label": { "string_id": "cfr-doorhanger-socialtracking-close-button" },
+            "event": "BLOCK"
+          },
+          {
+            "label": {
+              "string_id": "cfr-doorhanger-socialtracking-dont-show-again"
+            },
+            "action": { "type": "DISABLE_STP_DOORHANGERS" },
+            "event": "BLOCK"
+          }
+        ]
+      }
+    },
+    "targeting": "pageLoad >= 4 && firefoxVersion >= 71",
```

Comment:
Just a quick confirmation, are we sure about the target version as 71 and beyond?

Reply:
The messages are supposed to be always on. The version is a bit misleading. [In this commit](https://hg.mozilla.org/mozilla-central/rev/b656c8a33a948250362c32678582d902e421ddee) we renamed the trigger, because of performance issues we wanted the messages to no longer show on Firefox versions (71 & 72) that didn't get the perf improvements. I'll drop `firefoxVersion` from targeting because it will only make sense for 73+